lib/ansible/modules/systemd_service.py

```diff
@@ -484,17 +484,17 @@ def main():
             masked = out.strip() == "masked"
 
             if masked != module.params['masked']:
-                result['changed'] = True
-                if module.params['masked']:
-                    action = 'mask'
-                else:
-                    action = 'unmask'
+                action = "mask" if module.params['masked'] else "unmask"
```

Comment:
not sure why you changed this, ternary one liners are normally less clear than if/else